client/docs/swagger-ui/swagger.yaml

```diff
@@ -1716,6 +1716,43 @@ paths:
           format: uint64
       tags:
         - Query
+  /clerk/latest-state-sync-id:
```

Comment:
Shall we call this smth like `/clerk/event-records/latest-id` just to be compliant with the current structure?